CrossprojectpipingExternalModule.php

```diff
@@ -783,7 +783,7 @@ function getProjects() {
 			$dest_match_field_form = $Proj->metadata[$source_project['dest_match_field']]['form_name'];
 			foreach ($Proj->eventsForms as $eid => $formlist) {
 				# Quick-Fix for PHP8 Support
-				if (in_array($formlist, (array) $dest_match_field_form) !== false) {
+				if (in_array(implode("", (array) $dest_match_field_form), (array) $formlist) !== false) {
```

Comment:
For example if there are 2 forms: form_1 and form_2

`$formlist` is an array with value: ["form_1", "form_2"]
`$dest_match_field_form` is an array with value:
- ["f", "o", "r", "m", "_", "1"] in run 1
- ["f", "o", "r", "m", "_", "2"] in run 2

Therefore the check should be to implode `$dest_match_field_form` to be a string and check if `$formlist` contains it. Currently it's the other way around.

Reply:
@mmcev106  sure I'm on it

Reply:
Sure of what precisely?  My change, or yours?

Reply:
@mmcev106 I can confirm that your suggestion worked. Please update it to the changes and thank you for your verification. I was a bit confused about it. I believe your suggestion is more correct to the original intention of this check.

Reply:
> Sure of what precisely? My change, or yours?
I was testing on your change

Reply:
I tested again the edited version of your previous comment (with the updated `if` statement and the wrapper around the foreach loop). They all worked, for both cases where we have active form(s) defined and not defined.